### PR TITLE
delete key/value pair example

### PR DESCRIPTION
**Pull request by elinaawise**

### CODE REVIEW

File: docs/deleteKeyValuePair.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    username@instance deleteKeyValuePair> insert 567890 name first Joe
+    username@instance deleteKeyValuePair> insert 567890 name last Smith
+    username@instance deleteKeyValuePair> insert 567890 address city Columbia
+    username@instance deleteKeyValuePair> scan
+    567890 address:city []    Columbia
+    567890 name:first []    Joe
+    567890 name:last []    Smith
+    username@instance deleteKeyValuePair> flush -w
+    2019-04-18 11:01:47,444 [shell.Shell] INFO : Flush of table deleteKeyValuePair completed.
+
```
Comment (keith-turner) [inline docs/deleteKeyValuePair.md]: Could use `tables` command in the shell to get the table id.  Could also show how to scan the metadata table in the shell to see the list of files Accumulo is currently using.

```
scan -t accumulo.metadata -c file -r 2<
```

Above command assumes the table id is `2`.

File: docs/deleteKeyValuePair.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    username@instance deleteKeyValuePair> insert 567890 name last Smith
+    username@instance deleteKeyValuePair> insert 567890 address city Columbia
+    username@instance deleteKeyValuePair> scan
+    567890 address:city []    Columbia
+    567890 name:first []    Joe
+    567890 name:last []    Smith
+    username@instance deleteKeyValuePair> flush -w
+    2019-04-18 11:01:47,444 [shell.Shell] INFO : Flush of table deleteKeyValuePair completed.
+     
+To view the contents of RFile, we need to find the deleteKeyValuePair table name in Hadoop.
```
Comment (keith-turner) [inline docs/deleteKeyValuePair.md]: Could shorten this.

```
To view the RFile find the deleteKeyValuePair table in DFS.
```

File: docs/deleteKeyValuePair.md (hunk 1 above)
Comment (elinaawise) [reply docs/deleteKeyValuePair.md]: Thank you!  This is what I was looking for.

**Comment by keith-turner** [comment]
@elinaawise  I merged this in 34e3000377061b5827634506e918f51a56830fac with a few minor changes.  Thanks for the contribution.  If you would like to add yourself as a contributor, submit a PR for the website people page.

https://github.com/apache/accumulo-website/edit/master/pages/people.md

File: docs/deleteKeyValuePair.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+Compact the RFiles and verify the key-value pair was removed.  The new RFile will start with 'A'.
+
+```bash
+$ /path/to/accumulo shell -u username -p secret
+```
+    username@instance deleteKeyValuePair> compact -t deleteKeyValuePair -w
+    2019-04-17 08:17:15,468 [shell.Shell] INFO : Compacting table ...
+    2019-04-17 08:17:16,143 [shell.Shell] INFO : Compaction of table deleteKeyValuePair completed for given range
+
+```bash
```
Comment (keith-turner) [inline docs/deleteKeyValuePair.md:125]: This flush had no effect, so I removed it when I merged.

**Pull request closed without merging**